projects/plugins/jetpack/modules/masterbar/nudges/additional-css/class-atomic-additional-css-manager.php

```diff
@@ -38,10 +38,21 @@ public function __construct( $domain ) {
 	 * @param \WP_Customize_Manager $wp_customize_manager Core customize manager.
 	 */
 	public function register_nudge( \WP_Customize_Manager $wp_customize_manager ) {
+		$nudge_url  = $this->get_nudge_url();
+		$nudge_text = __( 'Purchase a Business Plan to<br> activate CSS customization', 'jetpack' );
+
+		if (
+			( defined( 'ENABLE_PRO_PLAN' ) && ENABLE_PRO_PLAN ) ||
+			! empty( $_GET['enable_pro_plan'] ) || // phpcs:ignore WordPress.Security.NonceVerification.Recommended -- Only used to customize output.
+			! empty( $_COOKIE['enable_pro_plan'] )
+		) {
+			$nudge_text = __( 'Purchase a Pro Plan to<br> activate CSS customization', 'jetpack' );
+			$nudge_url  = preg_replace( '/premium$/', 'pro', $nudge_url );
```

Comment:
Noting that I think this is wrong, but this is what is on WP.com right now. Approving to get things in sync; can resolve it in a follow-up since it's already seemingly broken to me.